build/debian-hyperkube-base/Makefile

```diff
@@ -24,10 +24,10 @@ ARCH?=amd64
 CACHEBUST?=1
 
 BASEIMAGE=gcr.io/google-containers/debian-base-$(ARCH):0.2
-CNI_RELEASE=0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff
+CNI_VERSION=v0.6.0
```

Comment:
we should have bumped the `debian-hyperkube-base` tag here.

Reply:
Will do in another PR.